en/news/_posts/2014-02-23-support-for-ruby-1-9-3-has-ended.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: news_post
+title: "Support for Ruby 1.9.3 has ended"
+author: "Olivier Lacan"
+translator:
+date: 2014-02-23 00:00:00 +0000
+lang: en
+---
+
+As of today, all support for Ruby 1.9.3 has ended. Bug and security fixes
+from more recent Ruby versions will no longer be backported to 1.9.3.
+
+This end of life was [announced over a year ago](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/).
+
+We highly recommend that you upgrade to Ruby 2.0 or above as soon as possible.
```

Comment:
@unak Where should people contact you. The website doesn't make it clear. http://bugs.ruby-lang.org?